Src/BootCamp.Chapter/InputMonitor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class InputMonitor : IInputMonitor<ConsoleKey>
+    {
+        public event EventHandler<InputPressedEventArgs<ConsoleKey>> InputPressed;
+
+        public void MonitorInput(ConsoleKey input)
```

Comment:
MonitorInput name is redundant. Monitor is enough (because the context and args on top of that explain what you are monitoring.